index.js

```diff
@@ -293,7 +242,8 @@ const generateRollupConfig = function(options) {
         name: settings.exportName,
         file: `dist/${settings.distName}.min.js`,
         format: 'umd'
-      }]
+        // remove .min.js output if should change watch is true
```

Comment:
I think its easier to understand that this happens if it happens right where we define it.